lib/features/copy-paste/ChoreoCopyPaste.js

```diff
@@ -0,0 +1,286 @@
+import ModelCloneHelper from 'bpmn-js/lib/util/model/ModelCloneHelper';
+import { getBusinessObject, is } from 'bpmn-js/lib/util/ModelUtil';
+import { assign, filter, forEach } from 'min-dash';
+import { getProperties, IGNORED_PROPERTIES } from 'bpmn-js/lib/util/model/ModelCloneUtils';
+
+/**
+ * This is copied from BpmnCopyPaste.js because it is not exported and could thus not be reused
+ * @param descriptor
+ * @param data
+ * @param properties
+ */
+function setProperties(descriptor, data, properties) {
+  forEach(properties, function(property) {
+    if (data[property] !== undefined) {
+      descriptor[property] = data[property];
+    }
+  });
+}
+
+/**
+ * This is copied from BpmnCopyPaste.js because it is not exported and could thus not be reused
+ * @param element
+ * @param properties
+ */
+function removeProperties(element, properties) {
+  forEach(properties, function(prop) {
+    if (element[prop]) {
+      delete element[prop];
+    }
+  });
+}
+
+/**
+ * Creates a descriptor for copying bpmn elements. A descriptor is an intermediary object that is later turned into a
+ * Shape object. (Potentially also Label object).
+ * This is copied from BpmnCopyPaste.js because it is not exported and could thus not be reused.
+ * This function is called on copy
+ * @param element
+ * @param descriptor an intermediary object that is later converted to a shape
+ * @returns {*}
+ */
+function bpmnDescriptor(element, descriptor) {
+  // Unfortunately the business object is not copied at this point. It will be cloned on paste
+  // this makes it necessary to copy some of the attributes already here and save them in the descriptor.
+  var businessObject = descriptor.oldBusinessObject = getBusinessObject(element);
+
+  var colors = {};
+
+  descriptor.type = element.type;
+
+  setProperties(descriptor, businessObject.di, ['isExpanded']);
+
+  setProperties(descriptor, element, ['isExpanded', 'collapsed']);
+
+  setProperties(colors, businessObject.di, ['fill', 'stroke']);
+
+  descriptor.colors = colors;
+
+  if (element.hidden) {
+    // hide elements that are currently hidden (e.g., elements in a collapsed SubChoreo or messages)
+    descriptor.hidden = element.hidden;
```

Comment:
Can we use the `setProperties` pattern from above here as well or does this need to be an if-clause?